tools/ci_build/build.py

```diff
@@ -517,12 +519,16 @@ def convert_arg_line_to_args(self, arg_line):
         "--armnn_home", help="Path to ArmNN home dir")
     parser.add_argument(
         "--armnn_libs", help="Path to ArmNN libraries")
+    parser.add_argument(
+        "--use_opencl", action="store_true",
+        help="Enable OpenCL Execution Provider for mobile GPU.")
     parser.add_argument(
         "--build_micro_benchmarks", action='store_true',
         help="Build ONNXRuntime micro-benchmarks.")
 
     # options to reduce binary size
-    parser.add_argument("--minimal_build", default=None, nargs='*', type=str.lower,
+    parser.add_argument("--minimal_build", default=None, nargs='?', type=str.lower,
+                        choices=["extended", "disabled"],
```

Comment:
what's the reason for this change? not understanding why we need `disabled` when not specifying the argument is the same isn't it?

Reply:
Ah, this is for enbale full aar build. When building android package, minbuild is enabled by default, `--minimal_build=disable` will do the trick. I can drop this commit tho.

Reply:
This unncessary change has been revert, as it breaks CI and documentation.